third_party/service-catalog/manifests/catalog/templates/apiregistration.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     name: release-name-catalog-apiserver
-  caBundle: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM5ekNDQWQrZ0F3SUJBZ0lSQVBPMjR1dUFLQXh4K0VwcDV5RjVDVFl3RFFZSktvWklodmNOQVFFTEJRQXcKRlRFVE1CRUdBMVVFQXhNS2MzWmpMV05oZEMxallUQWVGdzB4T1RBMU1ETXlNVFF4TWpoYUZ3MHlPVEEwTXpBeQpNVFF4TWpoYU1CVXhFekFSQmdOVkJBTVRDbk4yWXkxallYUXRZMkV3Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBCkE0SUJEd0F3Z2dFS0FvSUJBUUM5SkNpN250N2xUT2xiSEd0aDR5UjRDdmhyTGhXMXRPOVQ4ODk3SHB5dWpHbmUKaUVXUjd1a2VTZzh5d1Fzd0JQcVpMeWZxekd1VlhIMG1hYWs0aWJNSXJ2Ty9ZV0ZxUVk2ZUtKNzF1SU9vclB0eApPTTJYL3hNZGgvVlJvUTNib0JpYmhpcUVCZkxmWlV0cFFwWlBZMDlBRVgwbXZrM0daa3g5MU5qSmZqblFmdGxkCnV4ZTJrai8xUnovMlpHWWV6QjRpbm1VeVhsU3A1SW5wWkg3V1hGTEdUdUU3RVJydnBSNTk2bjVBaTJmTUJ6M2wKb3U2YURqUlI3QWIvN1FLbVJycjN6aU1ocXRweHl5SWsxT2NnREllNElwRWNWdUxzbWE3YXdCQmtocnRoV0RlbQppQ2VLZmNFM3hPS3NCZWdmUmFJdGJZclpOcVBZTHBPaUNLditISXhoQWdNQkFBR2pRakJBTUE0R0ExVWREd0VCCi93UUVBd0lDcERBZEJnTlZIU1VFRmpBVUJnZ3JCZ0VGQlFjREFRWUlLd1lCQlFVSEF3SXdEd1lEVlIwVEFRSC8KQkFVd0F3RUIvekFOQmdrcWhraUc5dzBCQVFzRkFBT0NBUUVBVWgyT3IyYzJUMVZJTnRNaWlQOUZtOFI1VUIrdAoxcU43azdyK3QwQ2o1cDBBVUhsc3ZIWnFLL2czYlp2R3U2Wlc5ZjJzQmwrODFIVTI4L2dKMXZLaG1kTFZDd25pCjRtUncxNFp3UGRKSnZlZXdsRW9RVFkxTlBlSC9PTURpdlVYL0djbGd6Umk5bDZHaEF3cWVtWHpnU09MazNBUXMKelBoNzJzc0ViMUtLUGVXNmZqVXRreEoydlo1MTN2dHM0WmxobGhURS9sOUFjNDBZbHNscWJlbmx1a0xwU2V2bQpyejZLeS9CK1pwSVhXK3BOVTZQc3VFRW1Qbkdpa2w3UGRtSXRpd0FKb0tvOVBoSlNRL0FLS2V5TWlnVnY2QitUCm11dXRNeWt1NVJ2ZGo0QytBdjNMQ0FkU3VyQlA1cDQwcEZoZ29JVkRiTkRMYytwU3VPWVErbW5ZM3c9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
+  caBundle: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM5ekNDQWQrZ0F3SUJBZ0lSQU0rSERZWTNKTjY1Z3BnN1Z0S0xaT1V3RFFZSktvWklodmNOQVFFTEJRQXcKRlRFVE1CRUdBMVVFQXhNS2MzWmpMV05oZEMxallUQWVGdzB4T1RBNE1Ea3lNRFF6TkRGYUZ3MHlPVEE0TURZeQpNRFF6TkRGYU1CVXhFekFSQmdOVkJBTVRDbk4yWXkxallYUXRZMkV3Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBCkE0SUJEd0F3Z2dFS0FvSUJBUUNyT2dxMWo2R0hBNjZSUHJicUNsN2kvL1NnbVdEbDgzYXNnNysybHZzTWI0SFkKdWxtYWE1bkRDa2J6US9XQlR0ZHlqVzNRcGFvV2UrU3crU0VTU0djbXdEcURsaVUyQjVyOFhKa1owTmtNVjBKYQpjUWtZNExlWDBGWGFKL2FvZ09IRjdsOHNZbkFzQ1FVUmdxaVFPUFNOcHN3Y2pnczg2ZjFObUtOYmRONDRNenB5Cnh0aldXeFNEUmltQW5JT0paditIQjlwSk1wTVVDZTJKUFhkYml4NXNhOEh1cEpwLzdCM3QzNG5WblFFcUdaYnMKMnpmclBaZjArUlJuanc5d0RlWnhTMjdUR0ZjQmNVU05KbUJmbytxS2NGdU91eElzN0l1SmYyYSs3NktwelE4QwoyaCsrU3FWdUw1SnZnRGo3aUlUWkNlWndVN28rd2pQdlVjclk2UjUvQWdNQkFBR2pRakJBTUE0R0ExVWREd0VCCi93UUVBd0lDcERBZEJnTlZIU1VFRmpBVUJnZ3JCZ0VGQlFjREFRWUlLd1lCQlFVSEF3SXdEd1lEVlIwVEFRSC8KQkFVd0F3RUIvekFOQmdrcWhraUc5dzBCQVFzRkFBT0NBUUVBSkpScEMva0RULzcrNzdIUXlkVGVzelBRRG9LZgphMzBlTmlSOVB4M0lyb01xd1NtRmxocEtlREFXa3JWRUxMMXJTNzJiQ3dmaU1sNVNRSmpJOU1hczcvdkU0SHcrCjJaaXh3NGhncEpGYXpaa0NCQjVaZmJZRUR6enhqbHFFK0VtaWl5UHVWS21obENBaGFlZjltVVAxUnhpdU9za3MKVWtuTHR6Nk5ubmg2U3k2THJIQzB1TWFieUp2S0QxNkdCK3ZONDd0aCtaeHREK2xabVhUdXFodklqR3JDUEFzQgpGUjRoS2Z5eWtqL0hQKzBUbkpPYVczOE5ZcmdSQXg5RTdjd2k4eHlnWElOY2hZcDM1K0JXVENhV1Zza2dWMDVDClF5eHczOXN1MU5zZzJ5TStUNnNVbDF3VzFnM1VJOVVDL29TNEMyWGhWck9rOE1tcmNQdnN3dWcwRlE9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
Why did this field and tls.crt change? Just curious

Reply:
it was the result of generating the new chart yamls, I think the certs changed because of the new PVC added